assemblies/assembly-ba-artificial-intelligence.adoc

```diff
@@ -0,0 +1,57 @@
+:context: artificial-intelligence
+
+= What is artificial intelligence?
+When you think about artificial intelligence (AI), machine learning and big data might come to mind. But machine learning is only part of the picture. Artificial intelligence encompasses the following technologies:
+* Robotics: The integration of technology, science, and engineering that produces machines that can perform physical tasks that are performed by humans
+* Machine learning: The ability of a collection of algorithms to learn or improve when exposed to data without being explicitly programmed to do so
+* Natural language processing: A subset of machine learning that processes human speech
+* Mathematical optimization: The use of conditions and constraints to find the optimal solution to a problem
+* Digital decisioning: The use of defined criteria, conditions, and a series of machine and human tasks to make decisions
+
+While science fiction is filled with what is referred to as pure AI, machines that perform better than people and cannot be distinguished from them and learn and evolve without human intervention or control, pure AI is decades away. Meanwhile, we have pragmatic AI which is a lot less frightening and much more useful to us today. Pragmatic AI is a collection of AI technologies that combined provide solutions to problems such as predicting customer behavior, providing automated customer service, and helping customers make purchasing decisions.
+
+Leading industry analysts report that previously organizations have struggled with AI technologies because they invested in the potential of AI rather than the reality of what AI can deliver today. AI projects were not productive and as a result investment in AI slowed and budgets for AI projects were reduced. Organizations are now seeing the practical reality of what AI can deliver. Being pragmatic means being practical and realistic.  A pragmatic approach to AI considers AI technologies that are available today, combines them where useful, and adds human intervention when needed to create solutions to real-world problems.
+
+.Pragmatic AI solution example
+A major transportation company has a fleet of trains and a network of train tracks. The company uses machine learning to identify geographic clusters where trains reported some type of problem. If a train reports a problem when it passes a specific position on the track it is recorded as data and if enough problems are recorded in the same location, you can identify a cluster. But that information is not complete enough to be useful so you can add digital decision to filter the data. For example, you can say that a failure must be a specific type of failure and must be reported by at least three trains. If that condition it met, you can use maths optimization to schedule a repair crew to investigate the track. Or if only one train has the same type of problem more than once at the location, you can use maths optimization to schedule the train for repair.
```

Comment:
> by at least three trains

by at least three different trains?

Reply:
This example sounds familiar :) can you kindly check internally it is enough of a **_general_** example, however, to be sure it's fine to describe it in this documentation, please?

Reply:
@matteo mortari who should I check with do you think? Myriam or someone else?

Reply:
I asked Edson to have a look in the jira. I wasn't able to tag him here.

Reply:
@emmurphy1 the example is technically correct, but it might be a bit hard for users to relate to it as it is very niche. There is no problem publishing the document as is for now, but I would recommend looking for an example that is more relatable to users and update the document in the future. @tarilabs , Daniele Zonca , and Tommaso Teofili can help with that.

Reply:
OK, thanks Edson.

Reply:
I agree that this example is quite specific, we can probably use another more general one related to customer support tickets:
- Customer files a support ticket (login error or another type of ticket)
- Machine Learning algorithm to classify ticket content to existing answers (based on keyword, NLP, etc)
- Apply digital decisioning to use consider multiple potential matches, apply additional logic (i.e. no more than 1 answer with score > threshold, etc) and produce the final output in terms of answer/forward to support

This three step pattern/approach can be used for many similar example: dispute, fraud detention, etc

Reply:
Thanks, @danielezonca. I updated the example. Please review.